README.md

```diff
@@ -13,6 +13,7 @@ Read and run the setup script:
 
 Load environment vars:
 ```
+unset $(grep --no-filename --invert-match '^#' .env.shared .env | cut -f1 -d"=" | xargs)
```

Comment:
sometimes we have to test run the scripts with a var unset. so might as well always start with a blank env.

Reply:
This is some advanced bash, impressive! I confirmed it works for me as well.